contracts/examples/digital-cash/src/lib.rs

```diff
@@ -0,0 +1,122 @@
+#![no_std]
+#![allow(unused_attributes)]
+
+use elrond_wasm::{imports, require, sc_error};
+imports!();
+
+mod deposit_info;
+use deposit_info::DepositInfo;
+
+pub const SECONDS_PER_ROUND: u64 = 6; 
+
+#[elrond_wasm_derive::contract(DepositImpl)]
+pub trait Deposit {
+	
+	#[init]
+	fn init(&self) {
+		let my_address: Address = self.get_caller();
```

Comment:
You don't need to store the owner in the contract storage anymore. You can simply call the API function: fn get_owner_address(&self) -> Address;

Reply:
Done!